### PR TITLE
interop-focused tweaks and temporary AAR exceptions

### PR DESCRIPTION
**Pull request by catkins-miso**
opening this on behalf of @getorymckeag to expedite the review

> * temporary ratings: AAR exceptions and temporary seasonal ratings.
> * Naming scheme, units headers gone
> * Geographic regions gone
> * HAL gone (and paging, by extension)

### CODE REVIEW

File: openapi.yaml
Comment (catkins-miso) [inline openapi.yaml:217]: More of a breadcrumb for later, and I think you probably have it covered with this schema, but I think we'll need some Ratings-Provider-level indication of the duration in minutes of each of these types. This goes to that "we don't want to model" friction with prepositioning.

Maybe the right solution here is to think of the names, i.e., normal, emergency, load shed like we've been talking about for the facility names, but the **duration** is the really important thing. PJM has four tiers--two emergency ones, short- and long-term, and MISO uses only one of them since the other doesn't accommodate our emergency duration. The RC could validate that there's an acceptable rating for each of their tiers. This provides flexibility down the road if a given Ratings Provider operates with different tiers; our tariff requires they give us a normal/emergency/load shed, not that they don't have stuff in between.

I've created an issue for this in TROLIE/spec#3

File: openapi.yaml
Comment (catkins-miso) [inline openapi.yaml]: is `limit-provenance` too precious or too imprecise? I think "where did the limit come from" is the exact purpose of this info object, so `provenance` fits, and I think `audit` implies once use of this info.

We're using `provenance` in other places and the description, so I take that to mean your conversations haven't been tripped up with other customers by using that term, which would be a good reason to use something more accessible.

We can change this later, it shouldn't hold up approval.

File: openapi.yaml
Comment (catkins-miso) [inline openapi.yaml:3114]: re: linting convo, some of these will be required and others not. This is obviously got to be there after creation but not on submission when it gets POST'd. I don't have the time atm to double check that we've addressed that, and perhaps it can wait. I'll let you be the judge.

**Review by catkins-miso — COMMENTED** [review]
@getorymckeag  consider this my approval of these changes with the caveat that I don't have the time this morning to load this up, lint it, etc. I also left some comments that you may consider before merging. I think we are well-aligned in principle on these changes, so I don't want to delay the merge.

File: openapi.yaml
Comment (getorymckeag) [reply openapi.yaml:217]: Agreed.  While the names are probably important for interop, as they are what everybody understands, it's kind of an awkward link back to each grid operators' (both TOs and RCs) operations manuals.  Under the hood, it might be better to think of these as "sustained", "1 hour", and "15 minutes".  Or whatever they map to, I just made those up.   While the API should have people's proper terminology from their manuals, it could be useful to understand the underlying meaning.

File: openapi.yaml
Comment (getorymckeag) [reply openapi.yaml]: Agreed, provenance is more consistent an accurate.  I think I was biased as I started thinking about this from the perspective of DYN calling it the "audit" data.  I changed it.

File: openapi.yaml
Comment (getorymckeag) [reply openapi.yaml:3114]: I think in general that I've been lazy about marking required fields in the spec, and I owe you some time cleaning that up.  I created an issue to track it- https://github.com/trolie/spec/issues/4

**Review by getorymckeag — APPROVED** [review]

**Pull request merged**
